test/io-tests/configs/unix-socket.config

```diff
@@ -1,4 +0,0 @@
-db-pool = 1
-server-unix-socket = "$(POSTGREST_TEST_SOCKET)"
-jwt-secret = "reallyreallyreallyreallyverysafe"
-db-config = false
```

Comment:
This test is not used anymore, now that we are using unix sockets in io tests everywhere.